src/main/kotlin/no/nav/familie/ef/sak/service/VurderingService.kt

```diff
@@ -66,11 +63,16 @@ class VurderingService(private val behandlingService: BehandlingService,
 
         val medlemskap = medlemskapMapper.tilDto(medlemskapsdetaljer = søknad.medlemskap,
                                                  pdlSøker = pdlSøker)
-
         val sivilstand = SivilstandMapper.tilDto(sivilstandsdetaljer = søknad.sivilstand,
                                                  pdlSøker = pdlSøker)
+        val vurderinger = hentVurderinger(behandlingId)
```

Comment:
Bra at denne flyttes ut. Skulle vi gjort tilsvarende for resten (Mapping)? Kan fort bli ganske stor.

Reply:
Er enig i at det kan være en god idé, Mattis. 
Noe alá dette ? `inngasvilkårGrunnlagMapper.tilDto(pdlSøker, søknad, ????)`

Reply:
Mulig det kommer i en senere PR